package.json

```diff
@@ -107,7 +106,7 @@
     "tslib": "^2.0.3",
     "tslint": "~5.11.0",
     "tslint-jasmine-rules": "^1.5.1",
-    "typescript": "~3.9.7",
+    "vercel": "^21.1.0",
```

Comment:
Did you intentionally remove the ts dep?

Reply:
No, missed this when rebasing, nice catch.

Reply:
fixed in c6e32aceb998dfac564e612816117695f36a62e9